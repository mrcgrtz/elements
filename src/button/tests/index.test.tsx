/**
 * Button Test
 * @author Marc GÃ¶rtz <https://marcgoertz.de/>
 */

import React from 'react';
import { render } from '@testing-library/react';
import Button from '..';

describe('<Button />', () => {
  it('renders a button', () => {
    const { container } = render(<Button />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
