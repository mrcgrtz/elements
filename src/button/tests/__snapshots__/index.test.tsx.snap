// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> renders a button 1`] = `
.c0 {
  display: inline-block;
  margin: 0;
  padding: 0.25rem 1rem;
  border: 0;
  font-size: inherit;
  font-family: system-ui,-apple-system,BlinkMacSystemFont,'Segoe UI',Roboto, Helvetica,Arial,sans-serif,'Apple Color Emoji','Segoe UI Emoji', 'Segoe UI Symbol';
  line-height: inherit;
  color: white;
  background-color: lightgrey;
  box-sizing: border-box;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  cursor: pointer;
}

.c0:hover,
.c0:focus {
  background-color: dodgerblue;
}

<button
  class="c0"
  type="button"
/>
`;
